nifi-nar-bundles/nifi-snmp-bundle/nifi-snmp-processors/pom.xml

```diff
@@ -20,8 +20,8 @@ language governing permissions and limitations under the License. -->
     <packaging>jar</packaging>
 
     <properties>
-        <snmp4j.version>1.10.1</snmp4j.version>
-        <snmp4j-agent.version>1.3.1</snmp4j-agent.version>
+        <snmp4j.version>2.5.6</snmp4j.version>
```

Comment:
As these appear to be the sole dependency changes, I went to check the LICENSE and NOTICE of snmp4j version, to see if the nar LICENSE and NOTICE needed to be changed.  It appears that the NOTICE will need to be updated (and was probably already incorrect (see https://nifi.apache.org/licensing-guide.html), and then those changes also reflected in the nifi-assembly NOTICE

```
  (ASLv2) SNMP4J 2
    The following NOTICE information applies:
       SNMP4J
       Copyright (C) 2003-2016  Frank Fock and Jochen Katz (SNMP4J.org)

  (ASLv2) Apache log4j
    The following NOTICE information applies:
      Apache log4j
      Copyright 2007 The Apache Software Foundation
```
Also, I don't know if they're due to changes you made or not, but I don't see commons-lang or commons-io as bundled dependencies, so they could be removed from the nar NOTICE